app/controllers/keywords_controller.rb

```diff
@@ -11,8 +11,24 @@ def index
     }
   end
 
+  def show
+    keyword = Keyword.find_by(id: params[:id])
+
+    if keyword.present?
```

Comment:
Instead of using find_by and checking present? can we just use `find`?

```
keyword = Keyword.find(params[:id])
```

Reply:
Yes we can but `find` will raise `ActiveRecord::RecordNotFound` if no record. If I want to render not found template using find then I have to rescue that raise, do you think is it better?

Reply:
FYI, the `NotFoundException` also render 404 screen on Production mode

Reply:
Do I need to add custom 404 page if I want to render my 404 page when app raise `NotFoundException` on production mode?

Reply:
How about if I want to render custom 404 page when app raise `NotFoundException` on production. As I know `NotFoundException` will render default 404 screen and also route to not existing page.
Please correct me if I'm wrong.

Reply:
yeah, first of all, if the people entering a non-existing `id` that sounds Not Found page, nothing weird at that point :) so we can go with the default behaviour like that.

Moving on custom the 404 page, you can [custom Rails error pages](https://www.google.com/search?sxsrf=ALeKk032ZZt5debZ8Ahp2zPW37RJLObwkg%3A1603166953537&ei=6WKOX4GzINDh-Aa0jLVw&q=custom+rails+errro+page&oq=custom+rails+errro+page&gs_lcp=CgZwc3ktYWIQAzIJCAAQyQMQFhAeMgYIABAWEB46BAgAEEdQtixYkS9ghDBoAHAEeACAAWOIAf8CkgEBNJgBAKABAaoBB2d3cy13aXrIAQjAAQE&sclient=psy-ab&ved=0ahUKEwjBy5-mpsLsAhXQMN4KHTRGDQ4Q4dUDCA0&uact=5)

Reply:
Fixed in 8a69ad397f51b935d6da91dad7f121383d406beb